src/alt/store/AltStore.js

```diff
@@ -55,7 +55,8 @@ class AltStore {
           return actionHandler.call(model, payload.data, payload.action)
         }, payload)
 
-        if (result !== false && !this.preventDefault) this.emitChange()
+        const shouldUpdate = this.StoreModel.config.shouldStoreUpdate && this.StoreModel.config.shouldStoreUpdate(this.state)
```

Comment:
How can you compare old state with new state here?
In React you can use `this.state` and `this.props` from `shouldComponentUpdate` to get the current state/props, and compare them with the new state/props, passed as arguments.